ui/src/containers/NodePageMetricsTab.js

```diff
@@ -110,24 +161,162 @@ const NodePageMetricsTab = (props) => {
+    const data = operateMetricRawData(avgStats[metricName][0]?.values);
+
+    let extra = {};
+    const metricType = typedAvgMetrics[metricName];
+    if (metricType !== undefined) extra.type = metricType;
+
     acc[metricName] = data.map((slot) => ({
       date: fromUnixTimestampToDate(slot[0]),
-      [metricType]: slot[1],
+      y: slot[1],
+      symbol: 'Cluster avg',
       ...extra,
     }));
     return acc;
   }, {});
 
+  let cpuData = nodeStatsData['cpuUsage'];
+  if (showAvg) cpuData = cpuData.concat(avgStatsData['cpuUsage']);
+  const tooltipConfigCPU = getTooltipConfig([
+    {
+      field: 'CPU Usage',
+      type: 'quantitative',
+      title: `CPU Usage - ${nodeName}`,
+      format: '.2f',
+    },
+    showAvg && {
+      field: 'cluster avg',
+      type: 'quantitative',
+      title: 'CPU Usage - Cluster avg',
+      format: '.2f',
+    },
+  ]);
+
+  let systemLoadData = nodeStatsData['systemLoad'];
+  if (showAvg)
+    systemLoadData = systemLoadData.concat(avgStatsData['systemLoad']);
+  const tooltipConfigSystemLoad = getTooltipConfig([
+    {
+      field: 'System Load',
+      type: 'quantitative',
+      title: `System Load - ${nodeName}`,
+      format: '.2f',
+    },
+    showAvg && {
+      field: 'cluster avg',
+      type: 'quantitative',
+      title: 'System Load - Cluster avg',
+      format: '.2f',
+    },
+  ]);
+
+  let memoryData = nodeStatsData['memory'];
+  if (showAvg) memoryData = memoryData.concat(avgStatsData['memory']);
+  const tooltipConfigMemory = getTooltipConfig([
+    {
+      field: 'Memory',
+      type: 'quantitative',
+      title: `Memory - ${nodeName}`,
+      format: '.2f',
+    },
+    showAvg && {
+      field: 'cluster avg',
+      type: 'quantitative',
+      title: 'Memory - Cluster avg',
+      format: '.2f',
+    },
+  ]);
+
+  const tooltipConfigIops = getTooltipConfig([
+    {
+      field: `Read`,
+      type: 'quantitative',
+      title: `Read - ${nodeName}`,
+      format: '.2f',
+    },
+    {
+      field: 'Write',
+      type: 'quantitative',
+      title: `Write - ${nodeName}`,
+      format: '.2f',
+    },
+    showAvg && {
+      field: 'read avg',
+      type: 'quantitative',
+      title: 'Read - Cluster avg',
+      format: '.2f',
+    },
+    showAvg && {
+      field: 'write avg',
+      type: 'quantitative',
+      title: 'Write - Cluster avg',
+      format: '.2f',
+    },
+  ]);
   // Combine the read/write, in/out into one dataset
-  const iopsData = nodeStatsData['iopsRead']?.concat(
-    nodeStatsData['iopsWrite'],
-  );
-  const controlPlaneNetworkBandwidthData = nodeStatsData[
+  let iopsData = nodeStatsData['iopsRead']?.concat(nodeStatsData['iopsWrite']);
+  if (showAvg)
+    iopsData = iopsData
+      .concat(avgStatsData['iopsRead'])
+      .concat(avgStatsData['iopsWrite']);
+
+  const tooltipConfigInOut = getTooltipConfig([
+    {
+      field: 'In',
+      type: 'quantitative',
+      title: `In - ${nodeName}`,
+      format: '.2f',
```

Comment:
```suggestion
      format: '.1f',
```
Please apply to all the tooltips.